python/metatensor-operations/metatensor/operations/_dispatch.py

```diff
@@ -715,3 +715,21 @@ def zeros_like(array, shape: Optional[List[int]] = None, requires_grad: bool = F
         return np.zeros_like(array, shape=shape, subok=False)
     else:
         raise TypeError(UNKNOWN_ARRAY_TYPE)
+
+
+def arange_like(array, start, end, requires_grad: bool = False):
```

Comment:
I don't see the point of requires_grad here, since this is intended to work with integer values, right?

Reply:
yes makes sense, I just copied pasted from zeros_like

Reply:
removed arange completely and used int_like as you suggested in another PR